src/cache-view.ts

```diff
@@ -69,7 +69,7 @@ export class ConfigurationWebview {
   // Save from the UI table into the CMake cache file if there are any unsaved edits.
   async persistCacheEntries() {
     if (this.dirty) {
-      telemetry.logEvent("editCMakeCache", {command: "saveCMakeCacheUI"});
+      await telemetry.logEvent("editCMakeCache", {command: "saveCMakeCacheUI"});
```

Comment:
Telemetry should be fire and forget.  I'm fixing `logEvent` so that it is not returning a Promise.